cmd/contour/shutdownmanager.go

```diff
@@ -30,8 +32,8 @@ import (
 )
 
 const (
-	prometheusURLFormat      = "http://localhost:%d/stats/prometheus"
-	healthcheckFailURLFormat = "http://localhost:%d/healthcheck/fail"
+	prometheusURLFormat      = "http://unix:%d/stats/prometheus"
```

Comment:
can we remove the `%d` from these? shouldnt need a port anymore

Reply:
👍